src/mk/def_rules.mk

```diff
@@ -847,13 +921,15 @@ define C_TO_MODULE_RECIPE =
 	@$(POSTCCOMPILE)
 endef
 
+# CRTCPPMOD is special because it launches PASE and can't be run in a PASE job so we 
+# spawn a job and lose the ability to get joblog info
 define CPP_TO_MODULE_RECIPE = 
 	$(MODULE_VARIABLES)
 	$(eval d = $($@_d))
 	@$(call echo_cmd,"=== Creating CPP module [$(notdir $<)] Note environment and library list are not set up")
 	$(eval crtcmd := crtcppmod module($(OBJLIB)/$(basename $(@F))) srcstmf('$<') $(CRTCMODFLAGS) $(ADHOCCRTFLAGS))
 	@$(PRESETUP) \
-	system "$(crtcmd)" >> $(LOGFILE) 2>&1 && $(call logSuccess,$@) || $(call logFail,$@)
+	$(SCRIPTSPATH)/launch "$(JOBLOGFILE)"  "$(crtcmd)" "Y" >> $(LOGFILE) 2>&1 && $(call logSuccess,$@) || $(call logFail,$@)
```

Comment:
Yes - I was looking for how we changed the code to avoid the PASE invoke .. but my project builds objects in all of these paths and all are working?  Perhaps I've missed something in merging my changes.  It would be good to get everything into an official build that I can re-install and then test with all of the projects I have 'BOB' enabled to flush out any remaining bugs.

Reply:
Right, I forgot, we enhanced the "launch" to take a parameter which is the "Y" above, so this is good, I don't know why  you didn't have that existing fix.